source/app/service-providers/fsal/util/search-file.ts

```diff
@@ -167,7 +167,9 @@ export default function searchFile (fileObject: MDFileDescriptor|CodeFileDescrip
   // Now immediately check if all required terms have matched. If not, we can
   // disregard this file now and save some computing time below.
   if (termsMatched !== termsToSearch.length) {
-    return []
```

Comment:
Did you make sure that NOT-operators disqualify a file? We have to make sure that we don't return *too many* results! That was the reasoning behind the fact that I returned an empty array here, and I'm not sure returning `finalResults` has the effects you think …?

Reply:
Yes, I didn't completely remember the code, just wanted to make sure that didn't get lost :)